esp-rom-sys/src/rom/mod.rs

```diff
@@ -0,0 +1,87 @@
+//! # ESP ROM libraries
+//!
+//! ## Overview
+//! The `rom` driver provides functionality related to the ROM (Read-Only
```

Comment:
I'd probably remove most of this, it's a bit waffly, the first couple of sentences summerize the crate well, so let's move it to lib.rs and cut the extra crc/md5 bits.